force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -260,12 +249,8 @@ export default class rd2EntryForm extends LightningElement {
     wiredRecurringDonationRecord(response) {
         if (response.data) {
             this.record = response.data;
-            this.header = editHeaderLabel + " " + this.record.fields.Name.value;
             this.isRecordReady = true;
             this.isEdit = true;
```

Comment:
Part of me wants to completely get rid of this flag, but it might be more work than it's worth. Could it become a getter that just checks `this.rd2State.recordId`, similar to `isCommitmentEdit`?

Reply:
I think that'll work, to go one step further though I'd like to drop this while `wiredRecurringDonationRecord` method.